<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="optionsBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Опции</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="initRadio">
       <property name="text">
        <string>Инициализация</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="editRadio">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Изменение a n-го или корня 
(для изменения старшего коэффициента: индекс = -1)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="evalRadio">
       <property name="text">
        <string>Вычисление значения полинома с заданным x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="resizeRadio">
       <property name="text">
        <string>Изменение размерности массива корней полинома</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="outputRadio">
       <property name="text">
        <string>Вывод полинома</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="outbutBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>961</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>320</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Выполнить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>130</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Окно инициализации полинома</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xEdit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>x = ...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idxEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>индекс</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="valEdit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>значение</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sizeEdit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>размер</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1008</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
